inc/saiport.h

```diff
@@ -47,6 +47,19 @@ typedef enum _sai_port_type_t
 } sai_port_type_t;
 
 /**
+* @brief Attribute data for #SAI_PORT_ATTR_BIND_MODE
+*/
+typedef enum _sai_port_bind_mode_t
+{
+    /** Port */
+    SAI_PORT_BIND_MODE_PORT,
+
+    /** Sub port */
+    SAI_PORT_BIND_MODE_SUB_PORT,
+
+} sai_port_bind_mode_t;
```

Comment:
I do not find any explanation for the bind mode in the v6 doc. what is mode_port, and what is sub_port?